tests/test_cli.py

```diff
@@ -44,37 +45,45 @@ def new_stack(self, image_version, keep_stacks, traffic, definition,
                     'status': 'CF:CREATE_COMPLETE',
                     'creation_time': '2016-01-01T12:00:00Z'}
 
-    def traffic(self, stack_id, percentage):
-        ...
-
     def wait_for_deployment(self, stack_id: str) -> [str]:
         return ['CF:WAITING', self.final_state]
 
-    def get_stacks(self) -> list:
+    def get_stacks(self, stack_reference=None) -> list:
```

Comment:
I think would be better to mock the payloads of HTTP call to lizzy agent than mocking the whole logic here. What do you think?